helm/krr-enforcer/templates/enforcer-cert-job.yaml

```diff
@@ -0,0 +1,191 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}-krr-enforcer-cert-job
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/component: krr-enforcer-cert-job
+  annotations:
+    helm.sh/hook: pre-install,pre-upgrade
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    helm.sh/hook-weight: "-5"
+spec:
+  backoffLimit: 3
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: krr-enforcer
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: krr-enforcer-cert-job
+    spec:
+      serviceAccountName: {{ .Release.Name }}-krr-enforcer-cert-job
+      restartPolicy: OnFailure
+      volumes:
+        - name: workdir
+          emptyDir: {}
+      containers:
+        - name: cert-job
+          image: "bitnami/kubectl:latest"
+          workingDir: /tmp/certs
+          volumeMounts:
+            - name: workdir
+              mountPath: /tmp/certs
+          command:
+            - /bin/bash
+            - -c
+            - |
+              set -e
+
+              # Generate a CA key and certificate
+              echo "Generating CA certificate..."
+              openssl genrsa -out ca.key 2048
+              openssl req -x509 -new -nodes -key ca.key -subj "/CN=robusta-krr-enforcer-ca" -days 365 -out ca.crt
```

Comment:
What happens when this expires?

Reply:
k8s admission calls will fail, and will be ignored.
The certificate is supposed to be renewed on every upgrade

Reply:
Is it possible to generatea certificate for longer than a year? This seems risky

Reply:
I'm going to add a note on the docs for it. 
I think adding automation for auto-detecting it, is an overkill right now